botogram/frozenbot.py

```diff
@@ -82,7 +83,7 @@ def __reduce__(self):
             self.validate_callback_signatures, self.process_backlog, self.lang,
             self.itself, self._commands_re, self._commands, self._chains,
             self._scheduler, self._main_component_id, self._bot_id,
-            self._shared_memory, self._update_processors,
+            self._shared_memory, self._update_processors, self.override_i18n
```

Comment:
Missing a comma after the last element (so it's easier to add new arguments here)